packages/likes/src/likes.tsx

```diff
@@ -127,21 +127,17 @@ function Likes({
                 className="like-kind">
                 <i className={likeVariant.icon}></i>
                 {variant === 'small' ? null : (
-                  <Heading6 className="osc-like-variant-label">
-                    {likeVariant.label}
-                  </Heading6>
+                    likeVariant.label
                 )}
               </section>
 
               {!hideCounters ? (
                 <section className="like-counter">
-                  <Paragraph>
```

Comment:
Waarom is deze `p` er nu uit?

Reply:
Styling van de NLDS paragraph ging niet lekker met de default styling van de button. En het is niet perse nodig, omdat alles al in een button element staat.